src/scroll/scroll_menu.py

```diff
@@ -32,14 +34,22 @@ def __init__(self, title: str, text: str, dir: str = CURRENT_WORK_DIR):
             padding=PADDING_WIDTH,
         )
 
+        def set_cancel() -> None:
+            """Cancel don't open file"""
+            self.future.set_result(None)
+
         def set_done() -> None:
             """Future object when done return None"""
+            if self.cur_file_path:
+                with open(self.cur_file_path, "r") as f:
+                    f_content = f.read()
+                text_editor.text_field.text = f_content
```

Comment:
It would be nice if we didn't do stuff like this. It is problematic for two reasons:
1. It creates a circular dependency between `text_editor` and `scroll_menu`.
2.  Directly mutating the state of `text_editor` is not a super clean pattern, we'd prefer to wrap this functionality in a method and expose that method to `scroll_menu`

Reply:
do you think text_editor should also be moved to maybe application state or just make the wrapper method?